packages/cli/src/commands/deploy/helper.ts

```diff
@@ -252,6 +251,29 @@ export const confirmOptionsUpdate = async (
   }
 }
 
+export const lookForManifestOptions = (
+  options: ManifestOption[] | undefined,
+) => {
+  if (options?.length === 0) {
+    return
+  }
+
+  const concatenatedOptions =
+    options !== undefined
+      ? options?.map((option) => option.name).join(', ')
+      : ''
+
+  console.log(cyan(bold('[info] Options found:')), green(concatenatedOptions))
+
+  console.log(
+    grey(
+      bold(
+        `[info] Please note that the option values won't be shared with a real Field but they'll be available only inside the Field Plugin Editor`,
```

Comment:
Do you think this message is ok, @eunjae-lee?

Reply:
Thanks for the feedback, @eunjae-lee. 🙌 

I'm going to adjust the text and also add the link to our `manifest documentation`, great addition. 💯